infra/modules/providers/azure/service-plan/README.md

```diff
@@ -0,0 +1,40 @@
+# Module Azure App Service Plan
+
+In App Service, an app runs in an App Service plan. An App Service plan defines a set of compute resources for a web app to run. These compute resources are analogous to the server farm in conventional web hosting. One or more apps can be configured to run on the same computing resources.
+
+This is a terraform module in Cobalt to provide an App Service Plan with the following characteristics:
+
+- Ability to specify resource group name in which the App Service Plan is deployed.
+- If a name is not specified, it will generate a random id and add it as a prefix for the names of all the resources created.
+- Ability to specify resource group location in which the App Service Plan is deployed.
+- Specify App Service Plan tier, size and kind of API manager to deploy
```

Comment:
Please provide the list of supported module settings including their description. Also include a reference to the terraform documentation https://www.terraform.io/docs/providers/azurerm/r/app_service_plan.html#argument-reference